framework/tests/src/tests/AnnotatedForTest.java

```diff
@@ -17,7 +17,7 @@ public AnnotatedForTest(File testFile) {
               "subtyping",
               "-Anomsgtext",
               "-Aquals=tests.util.SubQual,tests.util.SuperQual",
-              "-AuseSafeDefaultsForUnannotatedSourceCode");
+              "-AuseDefaultsForUncheckedCode=source,bytecode");
```

Comment:
I don't understand why unchecked code defaults should be turned of when compiling partially annotated libraries.  Could you please explain?